ethereumj-core/src/main/java/org/ethereum/manager/WorldManager.java

```diff
@@ -251,6 +257,103 @@ public void loadBlockchain() {
 */
     }
 
+    /**
+     * After introducing skipHistory in FastSync this method
+     * adds additional headerSource to Blockchain
+     * as Blockstore is incomplete in this mode
+     */
+    private void fastSyncDbJobs() {
```

Comment:
Seems WorldManager is not the best place for that code. Let's move it to separate class.